panic: nil pointer dereference when processing incoming image message

## Description

Whatomate crashes with a **nil pointer dereference** when receiving an incoming WhatsApp **image** message. The panic occurs in `chatbot_processor.go:134` inside `processIncomingMessageFull`.

## Stack Trace

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x100 pc=0x153019f]

goroutine 46 [running]:
github.com/shridarpatil/whatomate/internal/handlers.(*App).processIncomingMessageFull(...)
	/app/internal/handlers/chatbot_processor.go:134 +0x2ff
github.com/shridarpatil/whatomate/internal/handlers.(*App).processIncomingMessage(...)
	/app/internal/handlers/webhook.go:412 +0x385
created by github.com/shridarpatil/whatomate/internal/handlers.(*App).WebhookHandler in goroutine 45
	/app/internal/handlers/webhook.go:369 +0x121a
```

## Steps to Reproduce

1. Send an image message (without caption) to a WhatsApp number connected to Whatomate
2. The webhook receives the message with `type=image`
3. App panics immediately during processing

## Expected Behavior

Image messages should be handled gracefully — either processed or ignored with a log warning if not supported.

## Environment

- **Version:** v0.1.1-beta-0.14 (deployed on Railway)
- **Go runtime panic**, not a handled error

## Logs (before crash)

```
level=info message="Received message" from=55XXXXXXXXXXX type=image phone_number_id=XXXXXXXXX
level=info message="Processing incoming message" from=55XXXXXXXXXXX type=image profile_name=XXXXX
panic: runtime error: invalid memory address or nil pointer dereference
```

--- a/internal/handlers/chatbot_processor.go
+++ b/internal/handlers/chatbot_processor.go
@@ -128,7 +128,11 @@ func (a *App) processIncomingMessageFull(phoneNumberID string, msg IncomingTextM
 	}
 
 	// Get or create contact (always do this for all incoming messages)
-	contact, isNewContact, _ := contactutil.GetOrCreateContact(a.DB, account.OrganizationID, msg.From, profileName)
+	contact, isNewContact, err := contactutil.GetOrCreateContact(a.DB, account.OrganizationID, msg.From, profileName)
+	if err != nil {
+		a.Log.Error("Failed to get or create contact", "from", msg.From, "error", err)
+		return
+	}
 
 	// Store BSUID if provided and not already set
 	if msg.FromUserID != "" && contact.BSUID != msg.FromUserID {

